src/test/java/uk/gov/hmcts/reform/et/syaapi/service/PdfMapperServiceTest.java

```diff
@@ -0,0 +1,316 @@
+package uk.gov.hmcts.reform.et.syaapi.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.et.common.model.ccd.Address;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantOtherType;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantType;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantWorkAddressType;
+import uk.gov.hmcts.et.common.model.ccd.types.NewEmploymentType;
+import uk.gov.hmcts.et.common.model.ccd.types.RepresentedTypeC;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+import uk.gov.hmcts.reform.et.syaapi.constants.PdfMapperConstants;
+import uk.gov.hmcts.reform.et.syaapi.utils.ResourceLoader;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SuppressWarnings({"PMD.TooManyMethods"})
+class PdfMapperServiceTest {
+    private static final Integer TOTAL_VALUES = 56;
+    private PdfMapperService pdfMapperService;
+    private CaseData caseData;
+    private static final String YES = "Yes";
```

Comment:
Duplication of constants. Make the ones in PdfMapperService have default visibility then you can use them instead

Reply:
Found a YES and NO constant in et-common package so will use them